src/plugins/pbjx/components/revert-modal/RevertDetails.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Message from '@gdbots/pbj/Message';
+import RevertPropertiesTable from '../revert-properties-table';
+import { filterRevertableData } from '../../utils/filterData';
+import findNodeDiff from '../../utils/findNodeDiff';
+import fullMapsAndLists from '../../utils/fullMapsAndLists';
+
+const RevertDetails = ({ event, isFieldSelected, isDbValueSameAsNodeValue, onSelectField: handleSelectField }) => {
+  // find properties in node that were removed
+  const newNode = event.get('new_node').toObject();
```

Comment:
@rsumilang nit: this bit of code is duplicated a few times, could probably be consolidated into a util